src/cocotb/handle.py

```diff
@@ -1002,6 +1002,18 @@ def value(self) -> LogicArray:
     def value(self, value: LogicArray) -> None:
         self.set(value)
 
+    @deprecated(
+        "`int()` casts on handles have been deprecated. Use `int(handle.value)` instead."
```

Comment:
```suggestion
        "`int(handle)` casts have been deprecated. Use `int(handle.value)` instead."
```
A bit clearer IMO; but feel free to ignore.